templates/pytorch-1.7.x-python38/requirements.txt

```diff
@@ -0,0 +1,4 @@
+algorithmia>=1.0.0,<2.0
+six
+numpy>=1.16.0,<2.0
```

Comment:
Minor comment but numpy 1.16 is super old and I'm assuming this was just copy/pasted since forever ago.

Reply:
You're right, I'm going to update the numpy version and try validating this again; once we're done with the higher-prio work of the others first.